content/07.pathogenesis.md

```diff
@@ -46,22 +46,30 @@ One severe example includes reports of cytokine storm-like responses in patients
 
 #### Reproduction Number and Dynamics of Transmission
 
-Accurate estimates of the reproduction number of a virus are crucial to understanding the dynamics of infection and to predict the effects of different interventions.
-The basic reproduction number and the timescale of infection (measured by the infectious period and the exposed period) govern population-level epidemic dynamics, with R~0~ being one of most critical epidemiological parameters [@isbn:9780691116174].
-The basic reproduction number, R~0~, is the expected number of new (secondary) infections caused by one infected person, assuming no time dependence and a wholly susceptible population [@doi:10.1111/j.1467-9574.1996.tb01482.x].
-R~0~ is a unitless number which in mechanistic models is a combination of parameters principally related to the transmission rate (rate of infection transmitting interactions for an infectious individual) and the infectious period [@doi:10.1126/science.abb5659; @isbn:9780691116174].
-A pathogen can invade a susceptible population only if R~0~ > 1 [@isbn:9780691116174; @isbn:9780199209996].
-The effective reproduction number, R~t~, describes how the reproduction number may change over time, and is used to quantify deviations in R from R~0~ that would occur, for example, as some fraction of the population became infected or as interventions were put into place.
+Disease spread dynamics can be estimated using R~0~, the basic reproduction number, and R~t~, the effective reproduction number. 
+Thus, accurate estimates are crucial to understanding the dynamics of infection and to predict the effects of different interventions.
+R~0~ is one of most critical epidemiological parameters [@isbn:9780691116174]; R~0~ is the average number of new (secondary) infections caused by one infected person, assuming a wholly susceptible population [@doi:10.1111/j.1467-9574.1996.tb01482.x]. 
+A simple mechanistic model used to describe infectious disease dynamics is a susceptible-infected-recovered (SIR) state-transition model. 
+In this formulation individuals move through three states: susceptible, infected, and recovered; two parameters, $\gamma$ and $\beta$, are specified, the rate at which the infectious recover, and the infection transmission rate, respectively. 
+In this simple formulation, R~0~ is estimated as the ratio of $\beta$ and $\gamma$.[@doi:10.1126/science.abb5659; @isbn:9780691116174]. 
+A pathogen can invade a susceptible population if R~0~ > 1 [@isbn:9780691116174; @isbn:9780199209996].
```

Comment:
typo. You missed the "only" -- without the "only" then you leave open the wrong possibility that it can invade a susceptible population when R0<1 as well.
Correct sentence (as before):
"A pathogen can invade a susceptible population if R~0~ > 1 [@isbn:9780691116174; @isbn:9780199209996]."